sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_call_automation_client.py

```diff
@@ -166,6 +171,148 @@ def get_call_connection( # pylint: disable=client-method-missing-tracing-decorat
             **kwargs
         )
 
+    @overload
+    def connect_call(
+        self,
+        *,
+        server_call_id: str,
+        callback_url: str,
```

Comment:
this should go to positional parameter please check the beta5 branch and make the necessary changes